lib/custom-marked.js

```diff
@@ -0,0 +1,25 @@
+import marked from 'marked'
+
+/**
+ * render content with markup and custom first paragraph
+ *
+ * @param {string} content from dato
+ * @returns {string} formatted content in html
+ */
+
+export function renderMarkedContent (content) {
```

Comment:
put a default in front of function and you don't have to use the curly brackets in _staticPage.vue